webui/react/src/hooks/useModal/useModalUserSettings.tsx

```diff
@@ -24,6 +28,12 @@ const useModalUserSettings = (): ModalHooks => {
           <span className={css.label}>Username</span>
           <span className={css.value}>{username}</span>
         </div>
+        <div className={css.field}>
+          <span className={css.label}>Password</span>
+          <span className={css.value}>
+            <Button onClick={() => openChangePasswordModal()}>Change password</Button>
```

Comment:
nit: you should be able to pass the onClick without the callback function, e.g. `onClick={openChangePasswordModal}`

Reply:
i think type check requires a callback but changing it to a named function wrapped in useCallback